unstructured/ingest/cli/cmds/sharepoint.py

```diff
@@ -55,6 +58,24 @@ def add_cli_options(cmd: click.Command) -> None:
                     https://[tenant]-admin.sharepoint.com.\
                     This requires the app to be registered at a tenant level",
             ),
+            click.Option(
+                ["--permissions-application-id"],
+                default=None,
+                type=str,
+                help="Application id for ingesting permission (rbac) data",
+            ),
+            click.Option(
+                ["--permissions-client-cred"],
+                default=None,
+                type=str,
+                help="Credentials for ingesting permission (rbac) data",
+            ),
+            click.Option(
+                ["--permissions-tenant"],
+                default=None,
+                type=str,
+                help="Sharepoint permission (rbac) tenant name, such as: abcde.onmicrosoft.com",
```

Comment:
double checking: so there is an independent tenant specifically for permissions?